modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,204 @@
+----
+ oc delete managedcluster <cluster-name>
+----
+
+. Redo the <<custom-resource-creation,Custom Resource Creation>> section for this cluster.
+
+.Additional Resources
+
+Kubernetes Custom Resources explained within the context of Assisted Installer Service.
+
+|===
+| Resource | Explanation | Usage | Scoping | New Resource
+
+| BareMetalHost
+| Contains the connection information for the BMC of the target Bare Metal machine
+| Loads and boots the Discovery ISO on the target machine using the Redfish protocol.
+| Namespace-scoped
+| No*
+
+| InfraEnv
+| Contains information for pulling OCP onto the target Bare Metal machine
+| Used with ClusterDeployment to generate the Discovery ISO for the SNO cluster.
+| Namespace-scoped
+| Yes
+
+| AgentClusterInstall
+| Specifies the SNO cluster's configuration such as networking, number of supervisor (control plane) nodes, etc. Will show the kubeconfig and credentials after completed installation.
+| Used to specify the SNO cluster's configuration information and provide status during the installation of the cluster.
+| Namespace-scoped
+| Yes
+
+| ClusterDeployment
+| References the AgentClusterInstall to use.
+| Used with InfraEnv to generate the Discovery ISO for the SNO cluster.
+| Namespace-scoped
+| No*
+
+| NMStateConfig
+| Provides `MAC` \-> `IP` mapping and other network settings to the Assisted Installer. Not needed for DHCP.
+| Used to setup static IP for the SNO cluster's Kubernetes API server
+| Namespace-scoped
+| Yes
+
+| Agent
+| Contains hardware information about the target Bare Metal machine
+| Created automatically on the Hub once the Discovery ISO on the targeted machine is booted
+| Namespace-scoped
+| Yes
+
+| ManagedCluster
+| When a cluster will be managed by the Hub, it needs to be imported and known. This Kubernetes object provides that interface.
+| The Hub uses this resource to provision, manage and show status of clusters.
+| Cluster-scoped
+| No
+
+| KlusterletAddonConfig
+| Contains the list of services (provided by the Hub) to be deployed to a ManagedCluster
+| Tells the Hub which addon services to deploy to a ManagedCluster
+| Namespace-scoped
+| No
+
+| Namespace
+| Logical space for each ManagedCluster's resources existing on the Hub
+| Used to propagate resources to the ManagedCluster
+| Cluster-scoped
+| No
+
+| Secret
+| Three are created: BMC Secret, Private key Secret, and Image Pull Secret
+| *BMC Secret* is used to authenticate into the target Bare Metal machine using its username and password. *Private key Secret* is used to SSH into the target Bare Metal machine. *Image Pull Secret* contains authentication information for the ocp image installed on the target Bare Metal machine.
+| Namespace-scoped
+| No
+
+| ClusterImageSet
+| Contains OCP image information such as the repository and image name
+| Passed into resources to provide ocp images
+| Cluster-scoped
+| No
+|===
+
+NOTE: *These resources are new within the context of RHACM, not ocp.
```

Comment:
I think this note can be removed. also, FYI the note format we use is:

[NOTE]
====
note text
====

"ocp" or OCP are not used, use {product-title}